lib/anyway/testing.rb

```diff
@@ -2,7 +2,7 @@
 
 require "anyway/testing/helpers"
 
-if defined?(RSpec)
+if defined?(RSpec) && defined?(RSpec.configure)
```

Comment:
Let's check only for a constant:

```suggestion
if defined?(RSpec::Core)
```

Also, we can use `return unless defined?(...)` here.

Reply:
Yeah, changed on this one.